app/views/questions/show.html.erb

```diff
@@ -61,8 +61,10 @@
                 <p>Author: <%= answer.find_user %> Updated: <%= answer.updated_at.strftime("%D at %r") unless answer.updated_at.nil? %></p><br>
                 <br>
               </div>
-              <div id='answer-delete-button-admin' class='left card-action'>
-                <% if current_admin? %>
+              <div id='answer-delete-button' class='left card-action'>
+                <% if current_users_question?(@question) %>
+                  <%= button_to "Delete Answer","/answers/#{answer.id}", method: :delete, params: { id: answer.id}, class: "right red darken-4 btn-small" %></br>
```

Comment:
Good call on the refactor of the id for this div; it's more accurate to what it contains.